.github/workflows/shared/prompts/pydantic-ai-provider-mapping-sweep.md

```
@@ -62,12 +62,22 @@ be triggered by code you wrote and observed to fail.
 
 ## Deduplication — mandatory BEFORE filing an issue
 
-Search for existing issues using the MCP
-GitHub tools (not `gh` CLI — it's blocked by the firewall proxy):
+First check this sweep's own prior findings with a tight, server-side label
+filter — the `/search/issues` endpoint is blocked by the firewall proxy and
+there are no `mcp__github__*` tools, but the `?labels=` filter on the
+issue-list endpoint is allowed:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)